CHANGES/6795.feature

```diff
@@ -0,0 +1 @@
+Add variables so users can specify the names of each RPM package to install (pulp_pkg_pulpcore_name & pulp_install_plugins's pkg_name) or to just override the prefix (pulp_pkg_name_prefix).
```

Comment:
nitpick: can this line-wrap to 100 chars.

Reply:
Done, it's 2 lines now.